src/pages/event/[eventId]/status.js

```diff
@@ -51,6 +52,18 @@ const Status = () => {
   );
 };
 
-export const getServerSideProps = getApplicationServerSideProps();
+export const getServerSideProps = getApplicationServerSideProps(
+  async ({ req, query, cookies, queryClient }) => {
+    const { eventId } = query;
+    await useGetEventById({ req, queryClient, id: eventId });
+
+    return {
+      props: {
+        dehydratedState: dehydrate(queryClient),
+        cookies,
+      },
+    };
+  },
+);
```

Comment:
Can't comment on the `<Page>` component in this file because it wasn't changed, but: Events can also have the `StatusFormOnPage` scenario if their calendar type is `single`, `periodic` or `permanent`.

Reply:
Maybe better to do this in a follow-up PR as part of III-3576